api/v1beta1/hostedcluster_types.go

```diff
@@ -676,6 +675,18 @@ type KubevirtPlatformSpec struct {
 	// +optional
 	// +immutable
 	BaseDomainPassthrough *bool `json:"baseDomainPassthrough,omitempty"`
+	// ExtClusterKubeConfigSecret is the secret name that holds the kubeconfig for the external infra cluster
+	//
+	// +optional
+	// +immutable
+	ExtClusterKubeConfigSecret *KubeconfigSecretRef `json:"extClusterKubeConfigSecret,omitempty"`
+	// ExtClusterNamespace defines the namespace in the external infra cluster the virtual machines will be created at.
+	// If not specified, the namespace will be derived the same as if the hosted cluster is created at the management cluster, i.e.
+	// <hostedcluster_namespace>-<hostedcluster_name>
+	//
+	// +optional
+	// +immutable
+	ExtClusterNamespace string `json:"extClusterNamespace,omitempty"`
```

Comment:
Refactored the terminology throughout the code to use exclusively `InfraNamespace` and `InfraKubeconfig` and its variants (`InfraKubeConfigFile`, `InfraKubeConfigSecret`, etc.)

Reply:
nice, that definitely helps.  For the places in the code where we use the infra client, do you think we should explicility call it the `kubevirtInfraClient`.  We don't need that distiction for the API since we're already within the kubevirt platform struct.